src/test/java/org/broadinstitute/hellbender/tools/sv/SVCallRecordUtilsUnitTest.java

```diff
@@ -457,54 +474,99 @@ public Object[][] testCreateData() {
                             ALLELES_DEL, Lists.newArrayList(GENOTYPE_DEL_1, GENOTYPE_DEL_2), 1000, "+-",
                             GATKSVVCFConstants.StructuralVariantAnnotationType.DEL, Collections.singletonList(GATKSVVCFConstants.DEPTH_ALGORITHM),
                             null, null,
-                            TEST_ATTRIBUTES),
+                            TEST_ATTRIBUTES, -10.),
```

Comment:
This is QUAL = 100?

Reply:
Yes, we actually have our variant qualities scaled 0-999 still. In the future we will probably scale them down (we have already done so with GQs). I'm actually going to up this to `90.` to ensure there isn't a problem with that.